packages/network-map-layers/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "esnext",
+        "moduleResolution": "bundler",
+        "target": "ES2020",
+        "declaration": true,
+        "declarationMap": true,
+        "outDir": "./dist",
+        "strict": true,
+        "skipLibCheck": true,
+        "esModuleInterop": true,
+        "forceConsistentCasingInFileNames": true,
+        "allowSyntheticDefaultImports": true,
+        "noFallthroughCasesInSwitch": true,
+        "resolveJsonModule": true,
+        "noEmit": false
```

Comment:
Not the same options as the root tsconfig file ?

Reply:
It now use the same options except for jsx that it doesn't use.